.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ executors:
   node-executor:
     working_directory: ~/phovea
     docker:
-      - image: circleci/node:12.13-buster-browsers
+      - image: circleci/node:14.17-buster
```

Comment:
@anita-steiner @puehringer Why don't we switch directly to the most recent [LTS version Node 16](https://nodejs.org/en/about/releases/)?

Reply:
@thinkh One step at a time 😆 No, we knew that everything is compatible with Node 14 (some devs were using it for some time already), and ESLint required >= 14, such that it was the next logical step. We can of course evaluate Node 16 soon(ish), depending on how long TS4 and stuff like that takes. Maybe a test balloon would be nice beforehand?

Reply:
Yes, I agree. No rush. Maybe we put it on our internal agenda then.